### DISCUSSION TITLE
Build Bromite from source: Can't apply many patches because of outdated

### DISCUSSION THREAD

**Discussion by Jackychans** [bromite/bromite#1775] [Builds]
Hi,

I'd like to build Bromite from source chromium tag 96.0.4664.104. However, I tried to run to apply all patches and got many of them failed. For example patch Add-bookmark-import-export-actions.patch, which was produced in 2018 has file DownloadLocationDialogCoordinator.java with lots of differences compared to tag 96.0.4664.104. --> code conflict .

Any idea how to apply patch incrementially ? Just wanna release the build the same as the latest one

Thanks alot

**Comment by csagan5** [comment]
Latest version is 96.0.4664.183

They all apply correctly here.

**Reply by Jackychans** [reply]
Hi,

I'm able to patch all and  build android system webview without any error already, following [GN_ARGS](https://github.com/bromite/bromite/blob/master/build/GN_ARGS) correctly
However, the .apk file only about 45MB, significantly different to the working version. As a result, it cant be installed

Please look at the photos below and advise
![Screenshot 2022-02-08 114227](https://user-images.githubusercontent.com/6682672/152973853-af034f31-ddae-473f-b2f8-64eb63bdc1c8.png)
![Screenshot 2022-02-08 175737](https://user-images.githubusercontent.com/6682672/152973862-215b803b-8c4f-42cf-9198-d01673ce955f.png)

**Reply by Jackychans** [reply]
Later today, I tried running "gclient sync -D" found this . 

![Screenshot 2022-02-08 114227](https://user-images.githubusercontent.com/6682672/152987894-f4a9d2a1-cd54-4498-82d3-4b3d2260e2fd.png)



Not sure if this is the original issue

**Comment by Jackychans** [comment]
It's because of incorrect CPU variant. Now i can build without a hitch.

Thanks @csagan5. Anyway Could you help to answer me the question at https://github.com/bromite/bromite/discussions/1767 ?

Plus, I'd like to donate, could you share your email address ?

**Reply by csagan5** [reply]
There are donation instructions on the [official website](https://www.bromite.org/) and the [README](https://github.com/bromite/bromite/blob/master/README.md).
